src/components/ec-multiple-values-selection/ec-multiple-values-selection.vue

```diff
@@ -0,0 +1,275 @@
+            class="ec-multiple-values-selection__select-all--checkbox"
+            @change="toggleAll()"
+          />
+        </div>
+        <!-- TODO ONL-4911 the above is the select all checkbox option - visible only if isSelectAll is set to true -->
+        <li
+          v-for="item in selectedItems"
+          :key="item.value"
+          class="ec-multiple-values-selection__value-wrapper"
+          data-test="ec-multiple-values-selection__checkbox-deselect"
+        >
+          <ec-checkbox
+            checked
+            class="ec-multiple-values-selection__checkbox"
+            @checked-value-change="onDeselect(item)"
+          >
+            <template #label>
+              <div
+                class="ec-multiple-values-selection__label-wrapper"
+              >
+                <ec-icon
+                  v-if="item.icon"
+                  class="ec-multiple-values-selection__icon"
+                  :name="item.icon.name"
+                  :type="item.icon.type"
+                  :size="24"
+                />
+                <span class="ec-multiple-values-selection__label-text">{{ item.text }}</span>
+              </div>
+            </template>
+          </ec-checkbox>
+        </li>
+
+        <li
+          v-for="item in notSelectedItems"
+          :key="item.value"
+          class="ec-multiple-values-selection__value-wrapper"
+          data-test="ec-multiple-values-selection__value-wrapper"
+        >
+          <ec-checkbox
+            class="ec-multiple-values-selection__checkbox"
+            data-test="ec-multiple-values-selection__checkbox-select"
+            @checked-value-change="onSelect(item)"
+          >
+            <template #label>
+              <div
+                class="ec-multiple-values-selection__label-wrapper"
+              >
+                <ec-icon
+                  v-if="item.icon"
+                  class="ec-multiple-values-selection__icon"
+                  :name="item.icon.name"
+                  :type="item.icon.type"
+                  :size="24"
+                />
+                <span class="ec-multiple-values-selection__label-text">{{ item.text }}</span>
+              </div>
+            </template>
+          </ec-checkbox>
+        </li>
+      </div>
+    </ec-loading>
+  </div>
+</template>
+<script>
+// TODO ONL-4912 !!!don't forget to debounce search!!!
+import EcCheckbox from '../ec-checkbox';
+import EcIcon from '../ec-icon';
+import EcInputField from '../ec-input-field';
+import EcLoading from '../ec-loading';
+
+export default {
+  name: 'EcMultipleValuesSelection',
+  components: {
+    EcCheckbox, EcIcon, EcInputField, EcLoading,
+  },
+  model: {
+    prop: 'value',
+    event: 'change',
+  },
+  props: {
+    items: {
+      type: Array,
+      required: true,
+      default: () => ([]),
+    },
+    value: {
+      type: Array,
+      required: false,
+      default: () => ([]),
+    },
+    loading: {
+      type: Boolean,
```

Comment:
default:false for now? :show prop is required I think that what creates the issue

Reply:
The bug we have happens irrelevant of browser height as a I mentioned earlier.

The issue is that loading component overrides what popover tries to do. 
More in particular that's how popover hides itself . With `visibility: hidden` and `opacity: 0`. 

![Screenshot from 2020-11-12 16-19-37](https://user-images.githubusercontent.com/55188371/98968153-39387e80-2505-11eb-8dbc-c9425f4cbee3.png)

We must always set to something the `show` . If not the default visibility of the loading component will be this
![image](https://user-images.githubusercontent.com/55188371/98968663-de535700-2505-11eb-8313-44523c2dcc64.png)

As a result the loading will override the visibility of the popover. 
Why we don't see it? Because popover sets the opacity to 0 as well but this is still clickable.

so maybe for now default true with a comment?
`default: false // We will revert to false when we complete functionality`

Reply:
For now I've keep the loading and just unwrapped the content